src/fdc/MSXtar.cc

```diff
@@ -265,10 +293,12 @@ unsigned MSXtar::getNextSector(unsigned sector)
 }
 
 // get start cluster from a directory entry,
-// also takes care of BAD_FAT and EOF_FAT-range.
-static unsigned getStartCluster(const MSXDirEntry& entry)
+// BAD FAT and EOF FAT-range are normalized to a single EOF_FAT value (0xFFFFFFF)
```

Comment:
Detail. I would suggest to remove the numerical value `0xfffffff` from the comment. Or else write it as `0x0fffffff` (currently it's too easy to read it as all-1-bits).